src/pages/Folder/Folder.jsx

```diff
@@ -0,0 +1,72 @@
+import Header from "components/Header/Header";
+import Footer from "components/Footer/Footer";
+import styles from "./Folder.module.css";
+import AvatarImg from "assets/Avatar.png";
+import SearchIcon from "assets/Search.png";
+import { useEffect, useState } from "react";
+import CardList from "components/CardList/CardList";
+import { getFolderInfo } from "api/api";
+import { useAsync } from "hooks/useAsync";
+
+export default function Folder() {
+  const [search, setSearch] = useState("");
+  const [folderInfo, setFolderInfo] = useState({});
+  const [isLoading, loadingError, getFolderInfoAsync] = useAsync(getFolderInfo);
+
+  const handleSearchChange = (e) => {
```

Comment:
P2:
불필요한 매개변수는 지워주세요.

```suggestion
  const handleSearchChange = () => {
```